HomeTask-2/JSTask-2/partitionOn.js

```diff
@@ -0,0 +1,21 @@
+/*
+Problem link: http://www.codewars.com/kata/partition-on
+*/
+
+// partition the items array so that all values for which pred returns true are
+// at the end, returning the index of the first true value
+function partitionOn(pred, items) {
+  var evenItemsCopy = items.filter(pred);
+  var oddItemsCopy = items.filter(
+    function(value) {
+      return !pred(value);
```

Comment:
this can be written as items.filter(val => !pred(val))

Reply:
That looks good! I'll go through arrow functions documentation and will do the changes. Thank you.